photo-portfolio/src/main/java/com/throughapinhole/photoportfolio/Classes/Post.java

```diff
@@ -0,0 +1,81 @@
+package com.throughapinhole.photoportfolio.Classes;
```

Comment:
The convention is usually that you create an `Entity` for things that get into the database and a `DTO` that you send back to the client. This is an entity so I think it's best to have it in a `entities` package. `com.throughapinhole.photoportfolio.entities`. Also we usually don't create uppercase package names.

Reply:
Ok, sounds good. I'll change the file structure around a bit.

Reply:
Done - changed 'Classes' to 'entities'

Reply:
Merged your first branch also and re-arranged things in my two branches to match.